docs/conf.py

```diff
@@ -338,17 +339,3 @@
 
 # If false, no index is generated.
 #epub_use_index = True
-
-
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
```

Comment:
This should stop cpython docs from being included, which fixes https://github.com/bbcmicrobit/micropython/issues/729.